src/styles/pages/layout.scss

```diff
@@ -25,3 +25,4 @@
 @import "src/components/ui/Link/link.scss";
 @import "src/components/ui/SlideOver/slide-over.scss";
 @import "src/components/ui/SROnly/sr-only.scss";
+@import "src/components/ui/Toggle/toggle.scss";
```

Comment:
What do you think of creating a new section on this file to put these imports that aren't currently being used and leave them commented out? Considering we removed unused imports in the recent past, I'm not sure we should be added them back again.

Reply:
oh this can be removed for now, I just forgot to remove after creating the story